### DISCUSSION TITLE
Why are my network columns empty?

### DISCUSSION THREAD

**Discussion by charlesroper** [winsiderss/systeminformer#2008] [Questions / Answers] [answered]
My network columns don't seem to be working and I don't know why.

![SystemInformer_2024-03-20_200](https://github.com/winsiderss/systeminformer/assets/176/b7d159ee-bbe0-456b-9fe7-acf5040169e6)

I don't see anything in the Network tab of a process, either.

![SystemInformer_2024-03-20_198](https://github.com/winsiderss/systeminformer/assets/176/e8a8c330-f432-47fb-b840-5abfdce7f047)

I get nothing in stock Task Manager either.

However, in system information, I am seeing data.

![SystemInformer_2024-03-20_199](https://github.com/winsiderss/systeminformer/assets/176/c35d2756-7ca6-42b4-ad3b-622164729c95)

Does anyone know what could be the problem here?

Many thanks 🙏🏼

**Comment by Biswa96** [comment, marked as answer]
Try to run system informer as administrator.

**Reply by charlesroper** [reply]
Thank you, that worked. I assume there's no way of getting those network values without admin?

I feel foolish now because I did initially try running as admin using the elevate shield icon. I entered my admin username and password (it's a separate local admin account) and I assumed it had elevated. However, what it had done is silently fail and wasn't running as an administrator at all.

It turns out, the reason it did this is because I had System Informer in Google Drive as a portable app. When trying to elevate, the admin account couldn't find the streaming Google Drive (mapped to G:) so the elevation request didn't work.

I've installed it using the installer now and all is fine.

Thanks again.
